composer.json

```diff
@@ -0,0 +1,5 @@
+{
+    "require-dev": {
+        "mockery/mockery": "dev-master"
+    }
+}
```

Comment:
@nimid note, you can have 
```json
"autoload": {
    "classmap": ["omise/"],
    "exclude-from-classmap": ["/omise/libraries/"]
}
```
That you can get rid of maintaining `tests/autoload.php`, `function autoload($class) { ... }` function.